app/views/users/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<div><%= t(".title") %></div>
```

Comment:
Since this seems to be a title, it makes sense (and more semantic) to use a heading tag, rather than a generic div tag. Using header tags gives these element more value when bots inspect the page for SEO

Reply:
Cool, good to know! Done in sha: 1ea0d5e